src/styles/globals.css

```
@@ -88,8 +88,10 @@
   --color-background: var(--background);
   --color-foreground: var(--foreground);
 
-  --font-sans: var(--font-pretendard), ui-sans-serif, system-ui, sans-serif;
-  --font-mono: var(--font-pretendard), ui-sans-serif, system-ui, sans-serif;
+  --font-sans:
+    Pretendard, -apple-system, BlinkMacSystemFont, 'Apple SD Gothic Neo', 'Malgun Gothic', 'Nanum Gothic', sans-serif;
+  --font-mono:
+    Pretendard, -apple-system, BlinkMacSystemFont, 'Apple SD Gothic Neo', 'Malgun Gothic', 'Nanum Gothic', sans-serif;
```

Stylelint (17.12.0)

[error] Expected "Pretendard" to be "pretendard" (value-keyword-case)

(value-keyword-case)